### DISCUSSION TITLE
Implications on citizen developer?

### DISCUSSION THREAD

**Discussion by PS-asimansari** [microsoft/Power-Fx#168] [Q&A] [answered]
Hi,
I am a developer working exclusively on power platform for many years now. Its the only script that I have command on and I have made 100s of apps on it. I am actually very excited to learn how Powerfx can help me out in developing apps outside of Powerapps environment but I am unable to find anything really helpful.

**Comment by rajyraman** [comment, marked as answer]
@PS-asimansari - Have you looked [Acumatica Demo](https://www.youtube.com/watch?v=ik6k89WNjuk&t=1025s)? It shows you how they used Power Fx on their own application to perform calculations.

There is also [host samples](https://github.com/microsoft/power-fx-host-samples) that shows how to use Power Fx outside of Power Apps. Both of these, however, involve writing code in C#.

**Reply by PS-asimansari** [reply]
Thank you for your reply. I will look into acumatica and if i can learn anything there.

**Comment by MikeStall** [comment]
Releasing power fx open source here enables other developers to include power fx in their products. We also expect power fx to show up more broadly across the Power Platform. 
For citizen developers, this means that their power fx skills will continually become more useful as more products get fx-enabled.

**Reply by PS-asimansari** [reply]
Hmm. Yes that's good for us. Any idea if we would be able to develop PowerFX based applications in the near future?
